### PR TITLE
registering sidrakshe28

### PR DESCRIPTION
**Pull request by sidrakshe28**
kindly accept my request.

### CODE REVIEW

File: content/participants/ttjc@sidrakshe28/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+type: "participant"
+name: "Siddhika Rakshe"
+portfolio: 
+github-link: "https://github.com/sidrakshe28"
+linkedin-link: " https://www.linkedin.com/in/siddhika28/"
+looking-for: "internship"
+description: " well-versed with HTML|CSS|UI-UX|Android app development|learning React"
+---
+
+Hi! I am a Siddhika Rakshe, let me walk you through my intro, projects and blogs.
```
Comment (khusharth) [inline content/participants/ttjc@sidrakshe28/index.md:12]: Hey! @sidrakshe28 a small mistake here:
* I am **a** Siddhika ...

Comment (sidrakshe28) [reply content/participants/ttjc@sidrakshe28/index.md]: Sure .thanks for informing me.

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Comment by gkanishk** [comment]
Welcome onboard!! @sidrakshe28

**Pull request merged**